CMakeLists.txt

```diff
@@ -70,6 +70,14 @@ option(BUILD_SLIMGUI "Build the Qt5-based GUI for SLiM" OFF)
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c11 -Wno-attributes -Wunused-label -Wimplicit -Wunused-variable -Wunused-value -Wno-pragmas -Wempty-body -Wshadow -Wparentheses -Wmissing-prototypes -Wswitch -Wpointer-sign -Wsign-compare -Wstrict-prototypes -Wno-sign-conversion -Wuninitialized")
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -Wno-attributes -Wunused-label -Wunused-variable -Wunused-value -Wno-pragmas -Wempty-body -Wshadow -Wparentheses -Wswitch -Wsign-compare -Wno-sign-conversion -Wuninitialized")
 
+# Windows specific flags
+if(WIN32)
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D_POSIX_THREAD_SAFE_FUNCTIONS -DDO_MEMORY_CHECKS=0")
```

Comment:
I'm curious why you add `POSIX_THREAD_SAFE_FUNCTIONS` here.  SLiMgui is single-threaded, so I wouldn't expect this to be needed (and it might slow things down, since it probably introduces threading locks on functions or some such).

Reply:
Yes, something like that would be better.  Or you might be able to track down a header that provides `localtime_r` even when this -D is not defined.

Reply:
I just tested with `gnulib`'s `time_r`, and it works without the `POSIX_THREAD_SAFE_FUNCTIONS` flag now, so that is good.